README.md

```diff
@@ -30,92 +29,100 @@ Or install it yourself as:
+```bash
+$ rails g rest-api-generator:resource car name:string color:string
+```
 
-##### Index
-returns JSON instance of generated model from database with status ok
+Will generate following controller and the other files:
 
+```ruby
 
-create and update will have permitted params defined by the attributes defined in the command (except if the attribute is type reference)
-************************************************************************************
-### Spec
+class CarsController < ApplicationController
+  before_action :set_car, only: %i[show update destroy]
 
-##### Create
-checks if instance was saved in database
+  def index
+    @car = Car.all
+    render json: @car, status: :ok
+  end
 
-##### Update
-checks if instance was updated in database ok
+  def show
+    render json: @car, status: :ok
+  end
 
-##### Delete
-checks if instance was deleted from database
+  def create
+    @car = Car.create!(car_params)
+    render json: @car, status: :created
+  end
 
-##### Show
-check if status ok
+  def update
+    @car = Car.update!(car_params)
+    render json: @car, status: :ok
+  end
 
-##### Index
-check if status ok 
+  def destroy
+    @car.destroy!
+  end
 
-************************************************************************************
-### Factory
-The factory will be generated with it's custom generator, defined in the instalation of the factory-bot
+  private
 
-************************************************************************************
-### Routes
-the routes generated are the basic resources routes:
-    table_name: resources
+  def set_car
+    @car = Car.find(params[:id])
+  end
 
-************************************************************************************
-### Scope
-It's possible to add flag in the command to generate an endpoint with a parent
-   $ rails g generator table_name attributes --scope father_name
+  def car_params
+    params.require(:car).permit(:name, :color)
+  end
+end
 
-the only thing this won't genrate it's the nested routes, but it will generate all the previous files mentioned just liek a scaffold
+```
 
 ## Development
 
-After checking out the repo, run `bin/setup` to install dependencies. Then, run `rake spec` to run the tests. You can also run `bin/console` for an interactive prompt that will allow you to experiment.
+After checking out the repo, run `bin/setup` to install dependencies. Then, run `rake spec` to run the tests. You can
+also run `bin/console` for an interactive prompt that will allow you to experiment.
 
-To install this gem onto your local machine, run `bundle exec rake install`. To release a new version, update the version number in `version.rb`, and then run `bundle exec rake release`, which will create a git tag for the version, push git commits and the created tag, and push the `.gem` file to [rubygems.org](https://rubygems.org).
+To install this gem onto your local machine, run `bundle exec rake install`. To release a new version, update the
```

Comment:
To install this gem in your local machine, run `bundle exec rake install` and update the version number in `version.rb`, if you wish to release a new version. After that, run `bundle exec rake release`, which will create a git tag, push it alongside your commits and publish the `.gem` file into [rubygems.org](https://rubygems.org).

Reply:
This section was automatic generated, i will keep the same